File: fizzbuzz/FizzBuzz.java
public class FizzBuzz {

    public static void main(String[] args) {
        for(int i = 1;; i++) {
            String output = "";
            if(i % 3 == 0) {
                output += "Fizz";
            }
            if(i % 5 == 0) {
                output += "Buzz";
            }
            if(output.equals("")) {
                output = String.valueOf(i);
            }
            System.out.println(output);
        }
    }
    
}

File: fizzbuzz/FizzBuzz.cs
namespace FizzBuzz {
    class FizzBuzz {
    
    static void Main(string[] args) {
        for(int i = 1;true;i++) {
            string output = "";
            if(i % 3 == 0) {
                output += "Fizz";
            }
            if(i % 5 == 0) {
                 output += "Buzz";
            }

            Console.WriteLine((output != "") ? output : Convert.ToString(i));
        }   
    }
}
}

